src/zoo/owl_zoo_cmd.ml

```diff
@@ -113,17 +120,22 @@ let run_gist gist =
   run [|""|] tmp_script |> ignore
 
 
+let extend_dir gist =
+  let gid, vid, _ = Owl_zoo_dir.parse_gist_string gist in
+  dir ^ gid ^ "/" ^ vid
+
+
 let print_info () =
   let info =
     "Owl's Zoo System\n\n" ^
     "Usage: \n" ^
     "  owl [utop options] [script-file]\texecute an Owl script\n" ^
     "  owl -upload [gist-directory]\t\tupload code snippet to gist\n" ^
     "  owl -download [gist-id]\t\tdownload code snippet from gist\n" ^
-    "  owl -remove [gist-id]\t\t\tremove a cached gist\n" ^
+    "  owl -remove [-a] [gist-id]\t\t\tremove a cached gist\n" ^
```

Comment:
`-a` means all? Do you think we need this? What if I don't add `-a`, what the command will delete then?

Reply:
?